### ISSUE TITLE
show tables in kingshard only show the first node's table

### ISSUE THREAD

**Issue by csrgxtu** [flike/kingshard#367]
![image](https://user-images.githubusercontent.com/5053620/29549444-f217fe42-873a-11e7-8203-3c44e48d6bfc.png)

![image](https://user-images.githubusercontent.com/5053620/29549435-ddff1e4a-873a-11e7-9622-c2867bd8d015.png)

![image](https://user-images.githubusercontent.com/5053620/29549439-e775c6fe-873a-11e7-967a-b83a9c1d3e11.png)

hummmm, where are the tables on vps2?

**Comment by flike** [comment]
yes, kingshard now only show tables on vps1. But it can route your sqls in correct node. If you want check the tables in vps2, you can use sql hint(connect mysql with -c), and then send sql like this:
`/*node2*/show tables`
